src/EnergyPlus/DataSurfaces.cc

```diff
@@ -734,4 +734,43 @@ Real64 AbsBackSide(EnergyPlusData &state, int SurfNum)
     return AbsorptanceFromExteriorBackSide + AbsorptanceFromInteriorBackSide;
 }
 
+void GetVariableAbsorptanceSurfaceList(EnergyPlusData &state)
+{
+    if (!state.dataHeatBal->AnyVariableAbsorptance) return;
+    for (int surfNum : state.dataSurface->AllHTSurfaceList) {
+        auto const thisSurface = state.dataSurface->Surface(surfNum);
```

Comment:
This is why we need to eliminate all scalar uses of `auto`. So that we can check automatically that all non-iterator/non-lambda uses of `auto` are `auto &`.

Reply:
indeed. Just fixed this instance. I'll remember `&` in the future.